leaflet-providers.js

```diff
@@ -121,6 +121,12 @@
 						bounds: [[46.2, -5.5], [50, 0.7]]
 					}
 				}
```

Comment:
missing `,` here cf automated check fail ;)

Reply:
Thanks! Fixed in https://github.com/leaflet-extras/leaflet-providers/pull/562/commits/4b127491642883291f162c56e5a9389e115d59d7